packages/app-builder-lib/src/targets/MsiTarget.ts

```diff
@@ -70,17 +70,22 @@ export default class MsiTarget extends Target {
 
     const commonOptions = getEffectiveOptions(this.options, this.packager)
 
+    // wix 4.0.0.5512.2 doesn't support the arm64 architecture so default to x64 when building for arm64.
+    // This will result in an x64 MSI installer that installs an arm64 version of the application. This is a
+    // stopgap until the electron-builder-binaries wix version is upgraded to a version that supports arm64.
+    const wixArch = arch == Arch.arm64 ? Arch.x64 : arch;
```

Comment:
Thanks for the detailed comment! Would you mind also adding this line to the comment just so that it's easier to track when we can upgrade?
`// https://github.com/electron-userland/electron-builder/issues/6077`